packages/components/stories/SidePanel.js

```diff
@@ -162,11 +204,28 @@ stories
 								.fill('This is some random content')
 								.map((item, num) => <li key={num}>{item}</li>)}
 						</ol>
-						<IconsProvider defaultIcons={icons} />
 					</Layout>
 				);
 			}
 		}
 
-		return <WithLayout />;
+		return <WithLayout/>;
 	});
+
+apps.forEach(app => {
```

Comment:
You should add an example when the side panel is docked to show all the use case.

Reply:
You already have it [here](http://979.talend.surge.sh/containers/?selectedKind=HomeListViewExample&selectedStory=%F0%9F%8E%A8%20TDC%20default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)